event-adapter/pkg/types/install.go

```diff
@@ -13,18 +13,15 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
 limitations under the License.
 */
+//TODO Migrate the whole package to staging/src/k8s.io/api/ and pkg/apis/ as explained: https://github.com/kubernetes/community/blob/master/contributors/devel/api_changes.md#making-a-new-api-group
```

Comment:
I don't think you want to do this - I think you want to use instead existing package: https://github.com/kubernetes/kubernetes/tree/master/pkg/api